terraso_backend/apps/soil_id/models/depth_dependent_soil_data.py

```diff
@@ -23,3 +23,16 @@ class DepthDependentSoilData(BaseModel):
     soil_data_input = models.ForeignKey(SoilData, on_delete=models.CASCADE)
     depth_top = models.IntegerField(blank=True)
     depth_bottom = models.IntegerField(blank=True)
+    slope = models.IntegerField(blank=True)
+    texture = models.CharField(blank=True)
+    color_hue_float = models.FloatField(blank=True)
+    color_hue_char = models.CharField(blank=True)
+    color_value = models.FloatField(blank=True)
+    color_chroma = models.IntegerField(blank=True)
+    conductivity = models.FloatField(blank=True)
```

Comment:
for conductivity:
- there is a requirement to restrict this field (and some other fields) to a certain number of decimal places, so it might make sense to use the `DecimalField` (and associated `DecimalValidator`) to implement that restriction
- there are enums for conductivity testing method and units